mods/flowers/init.lua

```diff
@@ -31,6 +31,12 @@ minetest.register_alias("flowers:flower_viola", "flowers:viola")
 minetest.register_alias("flowers:flower_dandelion_white", "flowers:dandelion_white")
 
 
+local function on_flood(pos, oldnode, newnode)
+	minetest.add_item(pos, ItemStack(oldnode.name))
+	-- Remove flower
+	return false
```

Comment:
According to the docs you don't need to `return false` here.

Reply:
True fixed